service/provider/src/main/java/gov/va/vro/service/provider/camel/SaveToFileRoutes.java

```diff
@@ -18,21 +24,44 @@ class SaveToFileRoutes extends RouteBuilder {
   @Override
   public void configure() {
     if (config.persistTrackingEnabled) {
+      // checks if folder is writeable
+      logToFile("VRO started");
+
       // for v1
-      saveRequestToFile(PrimaryRoutes.INCOMING_CLAIM_WIRETAP);
-      saveRequestToFile(PrimaryRoutes.GENERATE_PDF_WIRETAP);
+      saveRequestToFile(PrimaryRoutes.INCOMING_CLAIM_WIRETAP, "claimSubmissionId");
+      saveRequestToFile(PrimaryRoutes.GENERATE_PDF_WIRETAP, "claimSubmissionId");
 
       // for v2
-      saveRequestToFile(MasIntegrationRoutes.MAS_CLAIM_WIRETAP);
-      saveRequestToFile(MasIntegrationRoutes.EXAM_ORDER_STATUS_WIRETAP);
+      saveRequestToFile(MasIntegrationRoutes.MAS_CLAIM_WIRETAP, "collectionId");
+      saveRequestToFile(MasIntegrationRoutes.EXAM_ORDER_STATUS_WIRETAP, "collectionId");
+    }
+  }
+
+  private void logToFile(String fileContents) {
+    DateFormat df = new SimpleDateFormat("yyyy-MM-dd-HHmmss");
+    String filename = df.format(Calendar.getInstance().getTime());
+    var path = Paths.get(config.baseTrackingFolder, filename);
+    try (PrintWriter printWriter = new PrintWriter(new FileWriter(path.toFile()))) {
+      printWriter.println(fileContents);
+    } catch (IOException e) {
+      log.error("Cannot write to " + config.baseTrackingFolder, e);
     }
   }
 
-  private void saveRequestToFile(String tapBasename) {
+  private void saveRequestToFile(String tapBasename, String idField) {
+    // TODO: make this reusable
     from(VroCamelUtils.wiretapConsumer("toFile", tapBasename))
+        // Good practice to clear the CamelRabbitmqExchangeName and CamelRabbitmqRoutingKey so it
+        // doesn't interfere with subsequent sending to rabbitmq endpoints
+        // https://camel.apache.org/components/3.19.x/rabbitmq-component.html#_troubleshooting_headers:
+        // > if the source queue has a routing key set in the headers, it will pass down to
+        // > the destination and not be overriden with the URI query parameters.
+        .removeHeaders("CamelRabbitmq*")
         .routeId("saveToFile-" + tapBasename)
-        .setHeader(Exchange.FILE_NAME, filepath("claimSubmissionId"))
-        .to("file:" + Paths.get(config.baseTrackingFolder, tapBasename));
+        .setHeader(Exchange.FILE_NAME, filepath(idField))
```

Comment:
`claimSubmissionId` doesn't exist for `v2` endpoints, which uses `collectionId` instead.